src/AmyrisBio/biolib.fs

```diff
@@ -30,121 +30,143 @@ module biolib =
+        [ (Ala, { trigram="Ala" ; letter = 'A' ; hydrophobicity = Some 0.620 })
+          (Arg, { trigram="Arg" ; letter = 'R' ; hydrophobicity = Some -2.530 })
+          (Asn, { trigram="Asn" ; letter = 'N' ; hydrophobicity = Some -0.780 })
+          (Asp, { trigram="Asp" ; letter = 'D' ; hydrophobicity = Some -0.900 })
+          (Cys, { trigram="Cys" ; letter = 'C' ; hydrophobicity = Some 0.290 })
+          (Gln, { trigram="Gln" ; letter = 'Q' ; hydrophobicity = Some -0.850 })
+          (Glu, { trigram="Glu" ; letter = 'E' ; hydrophobicity = Some -0.740 })
+          (Gly, { trigram="Gly" ; letter = 'G' ; hydrophobicity = Some 0.480 })
+          (His, { trigram="His" ; letter = 'H' ; hydrophobicity = Some -0.400 })
+          (Ile, { trigram="Ile" ; letter = 'I' ; hydrophobicity = Some 1.380 })
+          (Leu, { trigram="Leu" ; letter = 'L' ; hydrophobicity = Some 1.060 })
+          (Lys, { trigram="Lys" ; letter = 'K' ; hydrophobicity = Some -1.500 })
+          (Met, { trigram="Met" ; letter = 'M' ; hydrophobicity = Some 0.640 })
+          (Phe, { trigram="Phe" ; letter = 'F' ; hydrophobicity = Some 1.190 })
+          (Pro, { trigram="Pro" ; letter = 'P' ; hydrophobicity = Some 0.120 })
+          (Ser, { trigram="Ser" ; letter = 'S' ; hydrophobicity = Some -0.180 })
+          (Thr, { trigram="Thr" ; letter = 'T' ; hydrophobicity = Some -0.050 })
+          (Trp, { trigram="Trp" ; letter = 'W' ; hydrophobicity = Some 0.810 })
+          (Tyr, { trigram="Tyr" ; letter = 'Y' ; hydrophobicity = Some 0.260 })
+          (Val, { trigram="Val" ; letter = 'V' ; hydrophobicity = Some 1.080 })
+          (Sec, { trigram="Sec" ; letter = 'U' ; hydrophobicity = None }) // not documented int Eisenberg
+          (End, { trigram="End" ; letter = '*' ; hydrophobicity = None }) ]
+        |> Map.ofList
+
+    /// Translates amino acid represented with one character to a trigram, eg 'R' -> AminoAcidSymbol.Arg
+    let aaLetterToSymbol : char -> Result<AminoAcidSymbol, string> = function
+            | 'R' -> Ok Arg
+            | 'H' -> Ok His
+            | 'K' -> Ok Lys
+            | 'D' -> Ok Asp
+            | 'E' -> Ok Glu
+            | 'S' -> Ok Ser
+            | 'T' -> Ok Thr
+            | 'N' -> Ok Asn
+            | 'Q' -> Ok Gln
+            | 'C' -> Ok Cys
+            | 'U' -> Ok Sec
+            | 'G' -> Ok Gly
+            | 'P' -> Ok Pro
+            | 'A' -> Ok Ala
+            | 'V' -> Ok Val
+            | 'I' -> Ok Ile
+            | 'L' -> Ok Leu
+            | 'M' -> Ok Met
+            | 'F' -> Ok Phe
+            | 'Y' -> Ok Tyr
+            | 'W' -> Ok Trp
+            | '*' -> Ok End
+            | aa -> Error <| sprintf "ERROR: unknown amino acid letter '%c' in aaLetterToSymbolic" aa
+
+
+    /// Translates amino acid represented with trigram string to a trigram, eg "Arg" -> AminoAcidSymbol.Arg
+    let aaTrigramToSymbol : string -> Result<AminoAcidSymbol, string> = function
```

Comment:
please note, the name and return type change is obviously an API breaking change on DMX side, but i am happy to fix them in the future.

Reply:
No problem - I have only used this in a small script so far.  Just wanted to save the functionality for future code, so good to change now,

Reply:
cool, thanks!